documentation/user_docs/docs/example_scripts/dnd_vs_sqw.rst

```diff
@@ -0,0 +1,206 @@
+dnd vs sqw: the difference
+##########################
+
+Tutorial to illustrate and explain the different data types in Horace, and why
```

Comment:
Although you provide pointers to where the users might find what dnds and sqws are, you have to wade through a couple of sections of info before you start to talk about it. A basic paragraph on sqw is, dnd is to get this going looks needed